site/source/tutorials/005_form_widgets/index.md

```diff
@@ -1,26 +1,159 @@
+
+## Prerequisites
+You can [download](../assets/005_form_widgets-initial.zip) the demo project to get started.
+
+You also need to be familiar with TypeScript as Dojo 2 uses it extensively. For more information, refer to the [TypeScript and Dojo 2](../comingsoon.html) article.
+
+## Creating the form
+The first step to allowing the user to create new workers is to create a form. This form will contain the input elements that will accept the new worker's initial settings. To start, go to the `WorkerForm.ts` file and add this code:
+
+{% include_codefile 'demo/finished/biz-e-corp/src/widgets/WorkerForm.ts' lines:1-4,7-9,19-24,26-28,31,33,45-48,109-111 %}
+
+This widget will render an empty form with a `submit` handler that prevents the form from being submitted to the server. Before we continue to expand on this starting point though, let's get the form integrated into the application so that we can see the form develop as we add more features.
+
+To start, add the CSS rules that the widget is going to need. Include the following rules in the `workerForm.css` file.
+
+{% include_codefile 'demo/finished/biz-e-corp/src/styles/workerForm.css' lang:css %}
+
+Now, let's add the `WorkerForm` to the `App` class. Import the class and update the `render` method to draw it. It should be included after the `Banner` and before the `WorkerContainer` so the `render` method should look like this:
+
+{% include_codefile 'demo/finished/biz-e-corp/src/widgets/App.ts' lines:73,78-80,105-110 %}
+
+{% aside 'Reminder' %}
+If you can't see the application, remember to run `dojo build -w` to build the application and start the development server.
+{% endaside %}
+
+Now, open the application in a browser and inspect it via the browsers developer tools. Notice that the empty form element is being rendered onto the page as expected.
+
+## Adding form widgets
+Our form is going to contain fields to allow the user to enter the worker's first name, last name and e-mail address. It will also contain a save button that will use the form's data to create a new worker. We could create these fields and buttons using the `v` function to create simple virtual DOM elements. If we did this, however, we would have to handle details such as theming, internationalization ([i18n](https://en.wikipedia.org/wiki/Internationalization_and_localization)) and accessibility ([a11y](https://en.wikipedia.org/wiki/Accessibility)) ourselves. Instead, we are going to leverage some of Dojo 2's built-in widgets that have been designed with these things in mind. Add the following imports to `WorkerForm.ts`:
```

Comment:
"Our form is going to contain fields to allow the user to enter the worker's first name, last name and e-mail address." -> "Our form will contain fields allowing the user to enter the worker's first name, last name and e-mail address."